From 703a0e52af47847c724e1fe3a4d3527280613306 Mon Sep 17 00:00:00 2001
From: Sean Mackrory <mackrorysd@apache.org>
Date: Tue, 27 Mar 2018 15:51:44 -0600
Subject: [PATCH 2795/2863] HADOOP-15342. Updating ADLS connector to use the
 current SDK version (2.2.7). Contributed by Atul
 Sikaria.

(cherry picked from commit 9d7a9031a5978efc8d97566e35ebaace20db2353)

Change-Id: I7dfb24ade12757ecc59a196c0cb79e2b917dbe3b
(cherry picked from commit f3586c52c084cf738c23eda4e83e99681c73a04d)
---
 hadoop-tools/hadoop-azure-datalake/pom.xml |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-tools/hadoop-azure-datalake/pom.xml b/hadoop-tools/hadoop-azure-datalake/pom.xml
index b93f329..e4d6a44 100644
--- a/hadoop-tools/hadoop-azure-datalake/pom.xml
+++ b/hadoop-tools/hadoop-azure-datalake/pom.xml
@@ -34,7 +34,7 @@
     <minimalJsonVersion>0.9.1</minimalJsonVersion>
     <file.encoding>UTF-8</file.encoding>
     <downloadSources>true</downloadSources>
-    <azure.data.lake.store.sdk.version>2.2.5</azure.data.lake.store.sdk.version>
+    <azure.data.lake.store.sdk.version>2.2.7</azure.data.lake.store.sdk.version>
   </properties>
   <build>
     <plugins>
-- 
1.7.9.5

